projects/packages/post-list/src/class-post-list.php

```diff
@@ -57,16 +54,6 @@ public function register() {
 	 */
 	public function enqueue_scripts( $hook ) {
 		if ( 'edit.php' === $hook ) {
```

Comment:
It would be good to make sure that we load everything for Pages too. I'm not sure if this is the `$hook` for there.

We could even check the current post type and whether it has `thumbnail` support, so this will work for Woo products and other custom post types.

Reply:
I think edit.php is the correct hook. If we add the following additional filter and action to the code then the thumbnails appear in the Pages section too. Is that something we want to implement now? 

```
add_filter( 'manage_pages_columns', array( $this, 'add_thumbnail_column' ) );
add_action( 'manage_pages_custom_column', array( $this, 'populate_thumbnail_rows' ), 10, 2 );
```

Reply:
Yeah, let's. It would be good to have this enabled in both places.

Reply:
I'm cool with that, but should we be concerned with scope? Does it make sense that our Posts list package modifies Pages too?

Reply:
Pages are just another post type, and uses the posts list, so I think that's fine.

Reply:
Cool. Added and updated unit test.